Deploy-CAM.ps1

```diff
@@ -1503,6 +1502,9 @@ function New-ConnectionServiceDeployment() {
             -TenantId $tenantId `
             -ErrorAction Stop | Out-Null
 
+        Set-RadiusSettings `
```

Comment:
Is this in the admin context or the SP context?

Reply:
Looks like SP context

Reply:
Okay good. Thanks.

Reply:
Duh - that's set right above :)